Sources/BraintreeShopperInsights/BTShopperInsightsResult.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// A summary of the buyer's recommended payment methods.
 /// - Warning: This feature is in beta. It's public API may change or be removed in future releases.
-public struct BTShopperInsightsResult {
+public struct BTShopperInsightsResult: Sendable {
```

Comment:
Why does this result type need `Sendable` but not the other result types throughout the SDK?

Reply:
Is it because these have mutable properties and other VCs (not ShopperInsightsVC) conform to protocols that are marked @preconcurrency?